github/ci/prow-deploy/files/jobs/kubevirt/containerdisks/containerdisks-presubmits.yaml

```diff
@@ -263,6 +263,42 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: false
+    run_if_changed: "artifacts/fedora/.*"
+    annotations:
+      testgrid-create-test-group: "false"
+    cluster: prow-s390x-workloads
```

Comment:
This is only the difference between the already existing Fedora config?

Reply:
Everything is the same but the KUBEVIRTCI_TAG and KUBEVIRT_SLIM environment variables. These were originally included due to past issues or missing features in kubevirtci, but I kept them for consistency with other s390x jobs. Once this PR is approved and merged, I will clean everything up.

Reply:
Thanks. I see that there are also changes between labels and max concurrency.

Reply:
Oops, sorry I missed that one.